plugin/dnssec/metrics.go

```diff
@@ -0,0 +1,31 @@
+package dnssec
+
+import (
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// CacheSize is the number of elements in the dnssec cache.
+	CacheSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "dnssec",
+		Name:      "cache_size",
```

Comment:
"size" is ambiguous. Metric names should have their "unit" to make them understandable with minimal context.

I would suggest "cache_elements" to match the help text. Or is this entries? "cache_entries"